lib/vmdb/plugins.rb

```diff
@@ -104,6 +104,13 @@ def asset_paths
       end
     end
 
+    def server_role_paths
+      @server_role_paths ||= filter_map do |engine|
+        file = engine.root.join("content/server_roles.csv")
+        file if File.exist?(file.to_s)
+      end
```

Comment:
Does this need a `compact`

Reply:
Filter_map drops the `nil`s for you, this was a rubocop recommendation (replacing .map {}.compact with .filter_map {})